cluster/gce/util.sh

```diff
@@ -60,6 +67,11 @@ if [[ "${OS_DISTRIBUTION}" == "gci" ]]; then
   NODE_IMAGE_PROJECT=${KUBE_GCE_NODE_PROJECT:-google-containers}
```

Comment:
Ah, I did. But expanding it makes me think we should just remove the last two lines: lets not make any assumption about the `NODE_IMAGE` and `NODE_IMAGE_PROJECT` in this if block.